apps/google-analytics/src/Timeline.tsx

```diff
@@ -93,12 +89,20 @@ export default class Timeline extends React.Component<TimelineProps, TimelineSta
     }
 
     const { dimensions, start, end, pagePath, viewId } = this.props;
+    // pathWithoutTrailingSlash
+    let filters = `ga:pagePath==${pagePath}`
+
+    // filter to path with and without trailing slash if the slug doesn't end
+    // with a trailing slash
+    if (pagePath && !pagePath.endsWith('/')) {
+      filters = `${filters},ga:pagePath==${pagePath}/`
```

Comment:
Am I understanding correctly that the final `filters` string should contain a path with an ending slash and one without all the time?

Reply:
If there's a trailing slash, the filter will only be for the url with a trailing slash. If there's no trailing slash, then we'll filter for both the slashed and unslashed url. I could also just make it always filter for both. Not sure why I didn't do that.

Reply:
Makes sense to do both then.